packages/yuudachi/src/index.ts

```diff
@@ -33,6 +34,7 @@ const client = createClient({
 		GatewayIntentBits.GuildMessages,
 		GatewayIntentBits.GuildVoiceStates,
 		GatewayIntentBits.MessageContent,
+		AutoModActionExecutionIntentBits,
```

Comment:
```suggestion
		1 << 21, // AutoModActionExecution
```

We should just manually type this.